taca/analysis/analysis_nanopore.py

```diff
@@ -118,72 +165,90 @@ def get_original_samplesheet(run_id):
     return found_samplesheets[0]
 
 def parse_samplesheet(run_dir, lims_samplesheet):
-    # Parse Lims samplesheet into one suitable for nanoseq
+    """Parse Lims samplesheet into one suitable for nanoseq and anglerfish."""
     nanopore_kit = os.path.basename(lims_samplesheet).split('_')[0]
     nanoseq_samplesheet = os.path.join(run_dir, nanopore_kit + '_sample_sheet.csv')
-    content = 'sample,fastq,barcode,genome,transcriptome'
+    anglerfish_samplesheet = os.path.join(run_dir, 'anglerfish_sample_sheet.csv')
+    nanoseq_content = 'sample,fastq,barcode,genome,transcriptome'
+    anglerfish_content = ''
     with open(lims_samplesheet, 'r') as f:
-        for line in sorted(f.readlines()):
-            sample_name = line.split(',')[0]
-            nanoseq_barcode = line.split(',')[1]
+        lines = sorted(f.readlines())
+        first_sample_name = lines[0].split(',')[0] # Get name of first sample
+        fastq_location = os.path.join(run_dir, 'nanoseq_output', 'guppy', 'fastq') # Set the location of the first sample
+        for line in lines:
+            sample_name, nanoseq_barcode, run_type, illumina_barcode, location = line.split(',') #TODO remove location once/if removed in lims
             if nanoseq_barcode and nanoseq_barcode in BARCODES:
                 barcode = BARCODES[nanoseq_barcode]
+                is_pool = False
             else:
                 barcode = '0'
-            content += '\n' + sample_name + ',,' + barcode + ',,' # Only need sample and barcode for now.
+                is_pool = True
+            nanoseq_content += '\n' + sample_name + ',,' + barcode + ',,' # Only need sample and barcode for now.
+            if illumina_barcode:
+                # If there are no nanopore barcodes, the samples are from the same pool and will end up in
+                # the same nanoseq output file named after the firts sample in the sample sheet
+                if is_pool:
+                    anglerfish_content += sample_name + ',' + run_type + ',' + illumina_barcode + ',' + os.path.join(fastq_location, first_sample_name + '.fastq.gz') + '\n'
+                # If the samples are not the same pool, the nanoseq output is named by the barcode
+                else:
+                    anglerfish_content += sample_name + ',' + run_type + ',' + illumina_barcode + ',' + os.path.join(fastq_location, 'barcode' + barcode + '.fastq.gz') + '\n'
     with open(nanoseq_samplesheet, 'w') as f:
-        f.write(content)
-    return nanoseq_samplesheet
+        f.write(nanoseq_content)
+    if anglerfish_content:
+        with open(anglerfish_samplesheet, 'w') as f:
+            f.write(anglerfish_content)
+    return nanoseq_samplesheet, anglerfish_samplesheet
 
-def start_analysis_pipeline(run_dir, sample_sheet):
-    # start analysis detatched
+def start_nanoseq(run_dir, sample_sheet):
```

Comment:
Do you think there might be a need to modify these run parameters such as memory or similar? Could be fixed later though.

Reply:
Yeah, I think that's a good idea to maybe add some options so we can set the different parameters 👍 For now these should do for a while though I think, so I'll add it to my todo-list 🙂